src/Microsoft.Crank.PullRequestBot/Program.cs

```diff
@@ -572,30 +577,39 @@ private static async Task<IEnumerable<Result>> RunBenchmark(Command command)
                     foreach (var c in buildCommands) await ProcessUtil.RunAsync(ProcessUtil.GetScriptHost(), $"/c {c}", workingDirectory: cloneFolder, log: true);
 
                     Directory.SetCurrentDirectory(cloneFolder);
-                    RunCrank(_configuration.Defaults, benchmark.Arguments, profile.Arguments, buildArguments, $@"--json ""{workspace}base.json""");
+                    RunCrank(_configuration.Defaults, benchmark.Arguments, profile.Arguments, buildArguments, $@"--json ""{workspace}{BaseFilename}""");
 
                     await ProcessUtil.RunAsync("git", $@"fetch origin pull/{prNumber}/head", workingDirectory: cloneFolder, log: true);
-                    await ProcessUtil.RunAsync("git", $@"config --global user.name ""user""", workingDirectory: cloneFolder, log: true);
-                    await ProcessUtil.RunAsync("git", $@"config --global user.email ""user@company.com""", workingDirectory: cloneFolder, log: true);
+                    await ProcessUtil.RunAsync("git", $@"config user.name ""user""", workingDirectory: cloneFolder, log: true);
+                    await ProcessUtil.RunAsync("git", $@"config user.email ""user@company.com""", workingDirectory: cloneFolder, log: true);
                     await ProcessUtil.RunAsync("git", $@"merge FETCH_HEAD", workingDirectory: cloneFolder, log: true);
 
                     // Build head
                     foreach (var c in buildCommands) await ProcessUtil.RunAsync(ProcessUtil.GetScriptHost(), $"/c {c}", workingDirectory: cloneFolder, log: true);
 
                     Directory.SetCurrentDirectory(cloneFolder);
-                    RunCrank(_configuration.Defaults, benchmark.Arguments, profile.Arguments, buildArguments, $@"--json ""{workspace}head.json""");
+                    RunCrank(_configuration.Defaults, benchmark.Arguments, profile.Arguments, buildArguments, $@"--json ""{workspace}{HeadFilename}""");
 
                     // Compare benchmarks
-                    var result = RunCrank($"compare", $"{workspace}base.json", $"{workspace}head.json");
+                    var result = RunCrank($"compare", $"{workspace}{BaseFilename}", $"{workspace}{HeadFilename}");
 
                     results.Add(new Result(run.Profile, run.Benchmark, result));
                 }
                 finally
                 {
                     // Stop dotnet process
+                    // This assumes dotnet is part of the PATH, which might not work based on the host
```

Comment:
:D alright

Reply:
It's tentatively trying to clear the folder on the CI, but it fails that's fine too. We usually don't do that. But if you were to run it locally it would work.